packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -864,4 +863,45 @@ export default class SalesforceCRMService implements CRM {
 
     return query.records[0] as Record<string, any>;
   }
+
+  async findUserEmailFromLookupField(
+    attendeeEmail: string,
+    fieldName: string,
+    salesforceObject: SalesforceRecordEnum
+  ) {
+    const conn = await this.conn;
+
+    // Ensure the field exists on the record
+    const existingFields = await this.ensureFieldsExistOnObject([fieldName], salesforceObject);
+
+    if (!existingFields.length) return;
+
+    const lookupField = existingFields[0];
+
+    if (salesforceObject === SalesforceRecordEnum.ACCOUNT) {
```

Comment:
Only have functionality for account objects right now. We can always add logic for other types of records later.